frontend/components/TodayInHistory.tsx

```diff
@@ -1,59 +1,29 @@
-import axiosWithoutAuth from "@/axiosWithoutAuth";
 import { ModuleUnion, Topic } from "@/types/modules";
-import { Card, CardContent, CardHeader, Grid, Skeleton } from "@mui/material";
-import axios from "axios";
+import { Grid } from "@mui/material";
 import Link from "next/link";
-import { FC, useEffect, useState } from "react";
+import { FC } from "react";
 import ModuleUnionCard from "./cards/ModuleUnionCard";
 
-const TodayInHistory: FC = () => {
-  const [items, setItems] = useState<Exclude<ModuleUnion, Topic>[]>([]);
-  const [loading, setLoading] = useState(true);
-
-  useEffect(() => {
-    const cancelTokenSource = axios.CancelToken.source();
-    axiosWithoutAuth
-      .get("/api/home/today_in_history/", { cancelToken: cancelTokenSource.token })
-      .then((response) => {
-        setItems(response.data);
-        setLoading(false);
-      })
-      .catch((error) => {
-        if (axios.isCancel(error)) return;
-        console.error(error);
-        setLoading(false);
-      });
-    return () => {
-      cancelTokenSource.cancel("component unmounted");
-    };
-  }, [loading]);
+export interface TodayInHistoryProps {
+  modules: Exclude<ModuleUnion, Topic>[];
```

Comment:
We can replace every usage of `Exclude<ModuleUnion, Topic>` with `SerpModule`.
https://github.com/ModularHistory/modularhistory/blob/07eb6d7a1c2cb9735622ce30e396359e713f75fb/frontend/types/modules.d.ts#L136